holoviews/plotting/renderer.py

```diff
@@ -84,7 +84,7 @@ class Renderer(Exporter):
         The full, lowercase name of the rendering backend or third
         part plotting package used e.g 'matplotlib' or 'cairo'.""")
 
-    dpi=param.Integer(None, allow_None=True, doc="""
+    dpi=param.Integer(72, doc="""
```

Comment:
Do we want to be setting a specific dpi in the Renderer baseclass? That would suggest all backends will want to use this dpi...